package.json

```diff
@@ -39,7 +39,7 @@
     "@material-ui/styles": "^4.11.4",
     "@reach/auto-id": "^0.16.0",
     "@saleor/macaw-ui": "npm:@saleor/macaw-ui@0.7.4",
-    "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@1.1.3",
+    "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@0.0.0-pr-20240425162717",
```

Comment:
I'll replace this when https://github.com/saleor/macaw-ui/pull/793 is merged and released